### ISSUE TITLE
Microsoft JScript runtime error

### ISSUE THREAD

**Issue by JacksonBowe** [sst/sst#1524]
I'm going insane :)

I've followed these steps:
1. `npx create-serverless-stack@latest my-sst-app`
2. `npm install`
3. `npx sst start`
4. 
I then get hit with this:
![image](https://user-images.githubusercontent.com/26427343/158138820-f358d89d-5d35-4ffc-b5ca-1ea1812becf1.png)

I tracked the issue down and aparently it's because Windows Script Host doesn't support modern JS like `const` and `exports`
![image](https://user-images.githubusercontent.com/26427343/158138984-379a1673-6e1d-47c2-a7d3-b69ffde14d32.png)

What I've tried:
* Updating to latest version of java
* Updating to latest version of node (v16.14.0)
* Uninstalling aws-cdk (this works fine btw, I can `synth` and `deploy` my normal aws-cdk projects)
* Creating new projects in various locations
* Praying to a higher power
* Running these commands in cmd prompt
  1. regsvr32 jscript.dll and press ENTER.            
  2. regsvr32 vbscript.dll and press ENTER.

Is this a common error? My first time using SST

**Comment by jayair** [comment]
> Praying to a higher power

Oh wow, haha sorry you had to go through that.

Let me pull the team in on this. @fwang @thdxr

**Comment by JacksonBowe** [comment]
The higher power answered, and his name was Phil.

![image](https://user-images.githubusercontent.com/26427343/158269970-27e1962e-e85f-49fb-82dd-31fb1ef48963.png)

Clearer steps:
1. Create a JS file called whatever, eg `test.js`
2. Put a `console.log('1')` in it
3. Find it in Windows Explorer
4. Right click > Open with > Choose Another App
5. Tick always use this app to open .js files
6. More Apps > Scroll down > Choose another app on this pc
7. Navigate to node directory: eg C:\Program Files\nodejs
8. Select node.exe and __Open__
9. The icon for the file should change to the green node square

Above steps have stopped the JScript error from occurring.

There's a following error as follows but it may be unrelated to this so I will attempt to solve separately. 
```
PS D:\Projects\Mafia\mafia-sst> npx sst start
Look like you’re running sst for the first time in this directory. Please enter a stage name you’d like to use locally. Or hit enter to use the one based on your AWS credentials (Jackson): dev
Using stage: dev
Preparing your SST app

=======================
 Deploying debug stack
=======================

Usage: cdk -a <cdk-app> COMMAND

Commands:
  cdk.js list [STACKS..]             Lists all stacks in the app   [aliases: ls]
  cdk.js synthesize [STACKS..]       Synthesizes and prints the CloudFormation
                                     template for this stack    [aliases: synth]
  cdk.js bootstrap [ENVIRONMENTS..]  Deploys the CDK toolkit stack into an AWS
                                     environment
  cdk.js deploy [STACKS..]           Deploys the stack(s) named STACKS into your
                                     AWS account
  cdk.js watch [STACKS..]            Shortcut for 'deploy --watch'
  cdk.js destroy [STACKS..]          Destroy the stack(s) named STACKS
  cdk.js diff [STACKS..]             Compares the specified stack with the
                                     deployed stack or a local template file,
                                     and returns with status 1 if any difference
                                     is found
  cdk.js metadata [STACK]            Returns all metadata associated with this
                                     stack
  cdk.js acknowledge [ID]            Acknowledge a notice so that it does not
                                     show up anymore              [aliases: ack]
  cdk.js notices                     Returns a list of relevant notices
  cdk.js init [TEMPLATE]             Create a new, empty CDK project from a
                                     template.
  cdk.js context                     Manage cached context values
  cdk.js docs                        Opens the reference documentation in a
                                     browser                      [aliases: doc]
  cdk.js doctor                      Check your set-up for potential problems

Options:
  -a, --app                REQUIRED: command-line for executing your app or a
                           cloud assembly directory (e.g. "node bin/my-app.js")
                                                                        [string]
  -c, --context            Add contextual string parameter (KEY=VALUE)   [array]
  -p, --plugin             Name or path of a node package that extend the CDK
                           features. Can be specified multiple times     [array]
      --trace              Print trace for stack warnings              [boolean]
      --strict             Do not construct stacks with warnings       [boolean]
      --lookups            Perform context lookups (synthesis fails if this is
                           disabled and context lookups need to be performed)
                                                       [boolean] [default: true]
      --ignore-errors      Ignores synthesis errors, which will likely produce
                           an invalid output          [boolean] [default: false]
  -j, --json               Use JSON output instead of YAML when templates are
                           printed to STDOUT          [boolean] [default: false]
  -v, --verbose            Show debug logs (specify multiple times to increase
                           verbosity)                   [count] [default: false]
      --debug              Enable emission of additional debugging information,
                           such as creation stack traces of tokens
                                                      [boolean] [default: false]
      --profile            Use the indicated AWS profile as the default
                           environment                                  [string]
      --proxy              Use the indicated proxy. Will read from HTTPS_PROXY
                           environment variable if not specified        [string]
      --ca-bundle-path     Path to CA certificate to use when validating HTTPS
                           requests. Will read from AWS_CA_BUNDLE environment
                           variable if not specified                    [string]
  -i, --ec2creds           Force trying to fetch EC2 instance credentials.
                           Default: guess EC2 instance status          [boolean]
      --version-reporting  Include the "AWS::CDK::Metadata" resource in
                           synthesized templates (enabled by default)  [boolean]
      --path-metadata      Include "aws:cdk:path" CloudFormation metadata for
                           each resource (enabled by default)
                                                       [boolean] [default: true]
      --asset-metadata     Include "aws:asset:*" CloudFormation metadata for
                           resources that uses assets (enabled by default)
                                                       [boolean] [default: true]
  -r, --role-arn           ARN of Role to use when invoking CloudFormation
                                                                        [string]
      --staging            Copy assets to the output directory (use --no-staging
                           files with SAM CLI)         [boolean] [default: true]
  -o, --output             Emits the synthesized cloud assembly into a directory
                           (default: cdk.out)                           [string]
      --notices            Show relevant notices                       [boolean]
      --no-color           Removes colors and other style from console output
                                                      [boolean] [default: false]
      --version            Show version number                         [boolean]
  -h, --help               Show help                                   [boolean]

If your app has a single stack, there is no need to specify the stack name

If one of cdk.json or ~/.cdk.json exists, options specified there will be used
as defaults. Settings in cdk.json take precedence.

There was an error synthesizing your app.
PS D:\Projects\Mafia\mafia-sst>
```

**Comment by fwang** [comment]
Hey @JacksonBowe, I just released [v0.69.2](https://github.com/serverless-stack/serverless-stack/releases/tag/v0.69.2) with the fix. Let me know if it fixes the issue for you.

^^ To add some context, CDK made a recent change to their binary. It seemed to affect Windows users.

**Comment by thdxr** [comment]
We are transition this repo to https://github.com/sst/v2 and are closing this issue for now. If it is still relevant please reopen it there - sorry for the trouble
